w5e2.py

```diff
@@ -0,0 +1,37 @@
+#Cup of join
+''' Write a function called join, which receives an unlimited number of lists, each list as a parameter.
```

Comment:
Use """ and not '''  for docstrings, You dont have to translate the entire exercise

Reply:
"""